CMakeLists.txt

```diff
@@ -791,6 +791,8 @@ if (USE_BREAKPAD)
     endif()
 endif()
 
+option(USE_SYSTEM_LIBS "Tries to use system libs where possible." ON)
```

Comment:
This is not a good name since it really means to use any kind of precompiled one. How about inverting the meaning and calling it USE_SOURCE_LIBS? Meaning prefer to build stuff from source.

Reply:
Reverting it may not be a good idea if I change this to some `PREFER_SYSTEM_LIBS` with a fallback mechanism.

We can name it `USE_EXTERNAL_LIBS`, that name will fit both the system libraries and the ones from the `external_deps` folder.

Reply:
I implemented `PREFER_EXTERNAL_LIBS` instead.